src/glossary.md

```diff
@@ -57,6 +63,21 @@ can create such an lvalue without initializing it.
 Prelude, or The Rust Prelude, is a small collection of items - mostly traits - that are
 imported into very module of every crate. The traits in the prelude are pervasive.
 
+### Size
+
+The size of a value has two definitions.
+
+The first is that it is how much memory must be allocated to store that value.
+
+The second is that it is the offset in bytes between successive elements in an
+array with that item type.
+
+It is a multiple of the alignment, including zero. The size can change
+depending on compiler version (as new optimizations are made) and target
+platform (as `usize` varies).
```

Comment:
this is very very very minor, but i'd say "similar to how `usize` varies per-platform", as this wording (to me) makes it sound like it's based off of the size of usize.